reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -82,6 +82,7 @@ kamon.prometheus {
     # Hostname and port used by the embedded web server to publish the scraping enpoint.
     hostname = "0.0.0.0"
     port = 9095
+    metrics_path = "/metrics"
```

Comment:
Please call it `metrics-path`, we don't user underscore on any of our setting names.